nrf_security/CHANGELOG.rst

```diff
@@ -1,12 +0,0 @@
-.. _changelog_nrf_security:
```

Comment:
This erased CHANGELOG.rst is still referenced in the toctree in nrf_security

Reply:
I have amended the commit to remove this toctree reference, but this doesn't start the automatic build of this nor the referencing Prf #882. @thst-nordic Any Idea why?

Reply:
Cause that is not picked up by https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/882
I have re-triggered the build now.

Reply:
Great, marking as resolved